### ISSUE TITLE
What's the difference between this ranking and classify? 

### ISSUE THREAD

**Issue by feng-1985** [nyu-dl/dl4marco-bert#8]
The ranking loss function is the same as classification loss, what's makes it different?
In the training stage, the loss calculation didn't consider the query groups(multiple document share the same query).

**Comment by rodrigonogueira4** [comment]
The model is trained using a classification loss (cross entropy loss) to classify if a document is relevant or not for a given query. It outputs a probability of a document being relevant for a given query. The documents are re-ranked based on the probability assigned to each one of them.
Since the documents are classified independently, the model and the loss do not consider multiple documents.

**Comment by feng-1985** [comment]
Thanks for response, have you try the learning to rank?
